builds/misc/images-release.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         displayName: "Edge Agent Code Sign"
         inputs: 
           ConnectedServiceName: "Azure IoT Edge Code Sign 2"
-          FolderPath: $(Build.BinariesDirectory)/publish/Microsoft.Azure.Devices.Edge.Agent.Service
+          FolderPath: $(Build.BinariesDirectory)\publish\Microsoft.Azure.Devices.Edge.Agent.Service
```

Comment:
This task is in the Linux job, so revert back to forward slashes here. Only the Windows job will use the back slashes.

Reply:
Done